android/app/src/main/cpp/app/organicmaps/SearchEngine.cpp

```diff
@@ -111,7 +111,7 @@ jobject ToJavaResult(Result const & result, search::ProductInfo const & productI
   jni::TScopedLocalRef featureType(env, jni::ToJavaString(env, localizedType));
   jni::TScopedLocalRef address(env, jni::ToJavaString(env, result.GetAddress()));
   jni::TScopedLocalRef dist(env, ToJavaDistance(env, distance));
-  jni::TScopedLocalRef description(env, jni::ToJavaString(env, result.GetFeatureDescription()));
+  jni::TScopedLocalRef description(env, jni::ToJavaString(env, isFeature ? result.GetFeatureDescription() : ""));
```

Comment:
What about removing asserts from `GetFeatureDescription` and returning 
```
  enum class Type
  {
    Feature,  // as it is done now
    LatLon,   // return localized "Coordinate" text
    PureSuggest,  // return empty string
    SuggestFromFeature,  // return empty string
    Postcode  // return localized "Postal code" string
  };
```

@vng WDYT?

Reply:
The fix itself is ok. Can consider removing asserts in many search::Result functions later.